tests/helm/logs_otc/static/basic.output.yaml

```diff
@@ -105,8 +105,17 @@ data:
           overwrite_with: newest
           source_identifier: attributes["log.file.path"]
           type: recombine
+        - id: strip-trailing-newline
+          output: merge-multiline-logs
+          parse_from: body.log
+          parse_to: body
+          regex: |-
+            ^(?P<log>.*)
+            $
+          type: regex_parser
         - combine_field: body.log
-          combine_with: ""
+          combine_with: |2+
+
```

Comment:
This looks pretty weird, why does it get interpreted as a special character?

Reply:
This is just regular Yaml :wink: 

I looked it up and it seems to be a combination of the ["keep" chomping indicator](https://yaml.org/spec/1.2.2/#8112-block-chomping-indicator) `+` and the ["explicit" indentation indicator](https://yaml.org/spec/1.2.2/#8111-block-indentation-indicator) `2`.